geodata/models/__init__.py

```diff
@@ -3,4 +3,4 @@
 from .common import SpatialEntry  # noqa
 from .dataset import Dataset  # noqa
 from .geometry.base import GeometryArchive, GeometryEntry  # noqa
-from .raster.base import RasterFile, RasterEntry, ConvertedRasterFile  # noqa
+from .imagery.base import ImageFile, RasterEntry, ConvertedImageFile  # noqa
```

Comment:
We should add `ImageSet` to this so it is exposed at the top of the `geodata.models` namespace.